### PR TITLE
HPCC-15962 Envgen - add options to specify multiple esp instances

### PR DESCRIPTION
**Pull request by garonsky**
- Add cmd line flag -espnodes to specify the number of esp nodes the  wizard should generate

Signed-off-by: Gleb Aronsky gleb.aronsky@lexisnexis.com

### CODE REVIEW

**Comment by garonsky** [comment]
@Michael-Gardner please review

File: deployment/envgen/main.cpp [hunk 1]
```diff
@@ -83,6 +83,10 @@ void usage()
   puts("          for thor slaves. A node for thor master is automatically added. ");
   puts("          If not specified or specified as 0, no thor nodes");
   puts("          are generated");
+  puts("   -espnodes <number of esp nodes>: Number of nodes to be generated  ");
+  puts("          If not specified 1 esp node is generated");
+  puts("          If 0 is specified then all available nodes specified with assign_ips flag ");
+  puts("          will be used, others 0 esp nodes will be generated");
```
Comment (Michael-Gardner) [inline deployment/envgen/main.cpp]: @garonsky Looks good, we might want to add that espnodes only works for more than one instance if you explicitly declare assign_ips esp <range>

**Comment by richardkchapman** [comment]
@garonsky Can you respond to the review please

**Comment by garonsky** [comment]
Changes made based on feedback.
@richardkchapman this is ready to merge.

**Comment by richardkchapman** [comment]
@Michael-Gardner Please re-review

**Comment by Michael-Gardner** [comment]
@richardkchapman re-reviewed.  It's good to go.

**Pull request merged**